<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindFrame</class>
 <widget class="QFrame" name="FindFrame">
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="toolClose">
     <property name="icon">
      <iconset>
       <normaloff>:/resources/oxygen/16x16/actions/dialog_close.png</normaloff>:/resources/oxygen/16x16/actions/dialog_close.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editFind">
     <property name="inputMethodHints">
      <set>Qt::ImhNoAutoUppercase</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="toolPrevious">
     <property name="text">
      <string>Previous</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/resources/oxygen/16x16/actions/1leftarrow.png</normaloff>:/resources/oxygen/16x16/actions/1leftarrow.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="toolNext">
     <property name="text">
      <string>Next</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/resources/oxygen/16x16/actions/1rightarrow.png</normaloff>:/resources/oxygen/16x16/actions/1rightarrow.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkCase">
     <property name="text">
      <string>Case Sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkWholeWords">
     <property name="text">
      <string>Whole words</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelWrapped">
     <property name="text">
      <string>&lt;img src=&quot;:/trolltech/styles/commonstyle/images/fileinfo-16.png&quot;&gt;&amp;nbsp;Search wrapped</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
